helm/cluster-openstack/files/etc/gs-kube-proxy-patch.sh

```diff
@@ -9,7 +9,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-readonly dir="/run/kubeadm"
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+readonly dir
 
 # Run this script only if this is the init node.
 if [[ ! -f ${dir}/kubeadm.yaml ]]; then
```

Comment:
I understood this file was present only on `flatcar`. Would this work on `Ubuntu`? If the file is not present on `Ubuntu` the script will do nothing.